internal/connect/hwinfo.go

```diff
@@ -59,21 +62,20 @@ func getHwinfo() (hwinfo, error) {
 	hw.Cpus = result["cpus"].(int)
 	hw.Sockets = result["sockets"].(int)
 
-	hw.Hostname = hostname()
+	hw.Hostname = result["hostname"].(string)
```

Comment:
As noted before, leave a `TODO` note to tackle the possible error on casting here. I understand that we keep compatibility with the previous implementation, but it's actually quite bad.